packages/components/src/query-controls/index.tsx

```diff
@@ -96,12 +96,12 @@ export function QueryControls( {
 								value: 'date/asc',
 							},
 							{
-								/* translators: label for ordering posts by title in ascending order */
+								/* translators: Label for ordering posts by title in ascending order.translators: */
 								label: __( 'A → Z' ),
```

Comment:
I believe there is an accidental typo here where `translators:` is added at the end of the comment.

Reply:
Oh yes thank you